src/core/client/admin/components/BanModalQuery.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FunctionComponent } from "react";
+import { graphql } from "react-relay";
+import BanModal from "./BanModal";
```

Comment:
Can you please clean up and regroup the imports in this file?

Reply:
✅